AI-and-Analytics/Features-and-Functionality/IntelPython_GPU_numba-dpex_Genetic_Algorithm/README.md

```diff
@@ -22,103 +22,133 @@ In this sample, you will create and run the general genetic algorithm and optimi
-
-## Run the `Genetic Algorithms on GPU using Intel® Distribution of Python numba-dpex` Sample
-
-### On Linux*
-
-<!-- > **Note**: If you have not already done so, set up your CLI
-> environment by sourcing  the `setvars` script in the root of your oneAPI installation.
->
-> Linux*:
-> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
-> - For private installations: ` . ~/intel/oneapi/setvars.sh`
-> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
->
-> For more information on configuring environment variables, see *[Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html)*. -->
-
-#### Activate Conda
-
-1. Activate the Conda environment.
-   ```
-   conda activate base
-   ```
-   By default, the AI Kit is installed in the `/opt/intel/oneapi` folder and requires root privileges to manage it.
-
-   You can choose to activate Conda environment without root access. To bypass root access to manage your Conda environment, clone and activate your desired Conda environment using the following commands similar to the following.
-
-   ```
-   conda create --name usr_base --clone base
-   conda activate usr_base
-   ```
-
-#### Run the Python Script
-
-1. Change to the sample directory.
-2. Run the script.
-   ```
-   python IntelPython_GPU_numba-dpex_Genetic_Algorithm.py
-   ```
-
-#### Run the Jupyter Notebook (Optional)
-
-1. Launch Jupyter Notebook.
-   ```
-   jupyter notebook --ip=0.0.0.0
-   ```
-2. Follow the instructions to open the URL with the token in your browser.
-3. Locate and select the Notebook.
-   ```
-   IntelPython_GPU_numba-dpex_Genetic_Algorithm.ipynb
-   ```
-4. Run every cell in the Notebook in sequence.
-
-#### Troubleshooting
-
-If you receive an error message, troubleshoot the problem using the **Diagnostics Utility for Intel® oneAPI Toolkits**. The diagnostic utility provides configuration and system checks to help find missing dependencies, permissions errors, and other issues. See the [Diagnostics Utility for Intel® oneAPI Toolkits User Guide](https://www.intel.com/content/www/us/en/develop/documentation/diagnostic-utility-user-guide/top.html) for more information on using the utility.
+This sample code is implemented for GPUs using Python. The sample assumes you have numba-dpex installed inside a Conda environment, similar to what is installed with the Intel® Distribution for Python*.
+
+>**Note**: Read *[Get Started with the Intel® Distribution for Python*](https://www.intel.com/content/www/us/en/developer/articles/technical/get-started-with-intel-distribution-for-python.html)* to find out how you can achieve performance gains through Intel optimizations.
```

Comment:
Maybe we could add a link to this article into the Related Samples section? It seems logical to put it after we explained everything about this specific sample